_pages/index.md

```diff
@@ -3,98 +3,28 @@ title: Login.gov Dev Docs
+- Complete your SAML implementation. Review the minimum requirements for integration (API endpoints, certificates, SLO). Consider choosing a [SAML library]({{site.baseurl}}/saml_libs/) in the language and stack you're using.
+- [Configure your Service Provider]({{site.baseurl}}/configuring-your-sp/).
+- [Register your Service Provider]({{site.baseurl}}/registering-your-sp) with login.gov.
+- Test in our integration environment.
 
-**Logout service URL and Binding** — The single logout service URL is used to contact the Single logout profile (aka Single Logout Service). For example:
-
-```xml
-<SingleLogoutService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://idp.dev.login.gov/api/saml/logout" />
-```
-
-**x509 Public Certificate** — The public certificate is used to validate the authenticity of SAML requests received from login.gov. Minimum 2048 bits.
-
-### 2. Configuring Login.gov to accept your SP
-
-In order to successfully integrate, your SP application details must be whitelisted within Login.gov. We require the following details about your SP application:
-
-`*` — Denotes an attribute not found in the SAML WebSSO profile.
-
-| Attribute | Description |
-|-----------|-------------|
-| `issuer` | The SP entity ID — This is a unique string used to identify your SP with Login.gov. |
-| `acs_url` | Assertion consumer sign-on URL — This is your application endpoint which receives authentication assertions. |
-| `assertion_consumer_logout_service_url` | Assertion consumer logout service URL — This is the endpoint which receives logout requests and responses. |
-| `sp_initiated_login_url` * | This endpoint initializes authentication with Login.gov; it is used to trigger a new authentication request and response at the SP for better usability. |
-| `block_encryption` (optional) | This defines what type of encryption your SP supports. Currently, only `aes256-cbc` is supported. |
-| `cert` | Public certificate — The public certificate allows Login.gov to verify the authenticity of authentication and logout requests. |
-| `agency` * | This is used to group your SP to an agency, as well as inform the user about what agency the SP belongs. |
-| `friendly name` * (optional) | This is the user-friendly name for your SP application. |
-| `logo` * (optional) | This is your agency or SP's logo. This is used in the header of Login.gov when the user is authenticating to your SP. We recommend a semi-transparent PNG or SVG with a 3:2 width to height ratio at least 150px wide. |
-| `return_to_sp_url` * | Return-to SP URL — This is the URL of the SP which Login.gov provides to users when they wish to go directly to the SP site or cancel out of authentication. |
-| `attribute_bundle` * (optional) | The preset bundle of attributes your SP requires — We allow you to pre-define what attributes your SP requires, or you may request the attributes at run-time. |
-
-#### Sample configuration required by Login.gov:
-
-```yaml
```

Comment:
Related to the above, now that agencies will use the dashboard to register, they don't need to know the `yml` structure. _However_, it might be useful to show what complete SP details look like as an example, thoughts?